src/frontend/main.cpp

```diff
@@ -40,7 +52,10 @@ int main(int argc, char *argv[]) {
 
   instance.listen(name);
 
-  TopBar w;
+  TopBar w(parser.isSet(darkIcon));
   w.show();
+  if (parser.isSet(darkIcon)) {
```

Comment:
I think this should be `parser.isSet(startInTray)`

Reply:
oops 😔